tests/stat-paths-for-test.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+var path = require('path');
+var assert = require('assert');
+var statPathsFor = require('../lib/stat-paths-for');
+
+var fixturesPath = path.join(__dirname, 'fixtures');
+
+describe('statPathsFor', function() {
+  it('I am not 100% sure this is actually functioning correctly?', function() {
```

Comment:
I guess I'm not sure what this function is actually suppose to do... the `foo` module deps on `bar` and `linked`, but neither affects the results.